mmcv/ops/three_nn.py

```diff
@@ -0,0 +1,48 @@
+from typing import Tuple
+
+import torch
+from torch.autograd import Function
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext('_ext', ['three_nn_forward'])
+
+
+class ThreeNN(Function):
```

Comment:
suggest briefly introducing the method and giving a link to the paper in the docstring